contributors/devel/sig-architecture/validation-suites.md

```diff
@@ -0,0 +1,94 @@
+# Validation Suite Testing in Kubernetes
+
+A Kubernetes Validation suite is a subset of e2e tests that are used to
+validate correct functionality of an optional capability of Kubernetes. For example,
+Validation suites may be used to verify that optional storage or networking 
+capabilities are performing correctly.  They may also 
+be used to identify a group of tests that validate correct functionality on an
+optional platform such as Kuberenetes on Windows.  Kubernetes SIG groups are
+welcome to define Validation suites as needed.  Validation suites should not 
+be confused with [Conformance Tests](../sig-architecture/conformance-tests.md)
+which instead refers to the SIG Architecture core set of interoperable features that
+all conformant Kubernetes clusters must support. Instead, Validation suites 
+validate correct behavior on an optional or extension capability.  Over time a
+a Validation suite or portion of it may be promoted to become a 
+[Conformance Test](../sig-architecture/conformance-tests.md) if deemed necessary. 
+
+## Validation Suite Test Requirements
+
+Validation Test Suites may be used as deemed necessary by the associated Kubernetes
+SIG that is defining the Validation Suite. 
+
+## Running Validation Suite Tests
+
+Validation Suite tests are designed to be run even when there is no cloud provider
+configured. Validation Suite tests must be able to be run against clusters that have
+not been created with `hack/e2e.go`, just provide a kubeconfig with the
+appropriate endpoint and credentials.
+
+These commands are intended to be run within a kubernetes directory, either
+cloned from source, or extracted from release artifacts such as
+`kubernetes.tar.gz`. They assume you have a valid golang installation.
+
+```sh
+# ensure kubetest is installed
```

Comment:
34-48, we don't have to dictate exactly how exactly each test suite has to run. We just have to say that there should be 
- easy to use, follow patterns similar to existing e2e tests
- enough documenation
- ability to select some tests and skip others